_layouts/default.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html class="no-js">
   {% include head.html %}
-  <body>
+  <body class="path-support">
```

Comment:
Should we pull in a feature-parity `path-` helper? We know the full path at build time, and I know it's possible in Jekyll (though maybe doing it withing GH Pages sandbox rules might be tricky).